bedrock/firefox/templates/firefox/new/base.html

```diff
@@ -10,18 +10,33 @@
 
 {% block page_title_prefix %}{{_('Download Firefox')}} — {% endblock %}
 
-{# Bug 1438302 Avoid duplicate content for en-CA and en-GB pages. #}
-{%- block page_title -%}
-  {% if LANG == 'en-CA' %}
+
+{% if switch('firefox_new_title_experiment') %}
+    {# Bug 1438302 Avoid duplicate content for en-CA and en-GB pages. #}
+    {%- block page_title -%}
+    {% if LANG == 'en-CA' %}
+    The Fast and Free Web Browser 🦊 (CA)
+    {% elif LANG == 'en-GB' %}
+    The Fast and Free Web Browser 🦊 (UK)
+    {% else %}
+    {{_('The Fast and Free Web Browser 🦊')}}
```

Comment:
This is breaking translations, you'll need to use the existing `Free Web Browser` string as a fallback.